src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/MatchScoreClient.kt

```diff
@@ -57,7 +59,7 @@ data class MatchRecord(
         firstName = personEntity.firstName,
         lastname = personEntity.firstName,
         dateOfBirth = personEntity.dateOfBirth?.toString(),
-        pnc = personEntity.references.firstOrNull { it.identifierType == IdentifierType.PNC }?.identifierValue,
+        pnc = personEntity.references.getType(IdentifierType.PNC).firstOrNull()?.identifierValue ?: "",
```

Comment:
Can we write a test to prove that this is necessary?
Also maybe consider a convenience function like `personEntity.getPNC()`